import-automation/cloudbuild/cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+- name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  args: ['import-automation/cloudbuild/task.sh']
+  env:
+  - 'COMMIT_SHA=$COMMIT_SHA'
```

Comment:
I worry about using environment variables to pass around values.  Explicitly using command-line arguments is better because it is explicit.  They are also type checked and have default values.  However, sometimes they are necessary in some systems.

Reply:
The main reason I chose to use environmental variables was that I did not want to add another dependency as Cloud Build would have to install it every time.

Reply:
It is still worrisome, but I defer.